service/src/main/java/com/zebrunner/reporting/service/RabbitMQProperties.java

```diff
@@ -0,0 +1,17 @@
+package com.zebrunner.reporting.service;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@Getter
+@Setter
+@ConfigurationProperties(prefix = "spring.rabbitmq")
+public class RabbitMQProperties {
```

Comment:
Let's wire and use this: **org.springframework.boot.autoconfigure.amqp.RabbitProperties**.